src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id)
+                .orElseThrow(() -> {
+                    return new DataProcessingException("CinemaHall not found for id  " + id,
+                            new Exception());
+                });
```

Comment:
1. DataProcessingException is for problems with DB, when we don't have any result, operation just failed.
When we have fine answer from DB and just don't have requested object there exception should be different
2. orElseThrow structure should be simplified
```suggestion
                .orElseThrow(() -> new EntityNotFoundException("CinemaHall not found for id  " + id));
```